tools/reviewer/job/sync/LineNumberConverter.java

```diff
@@ -164,7 +164,21 @@ private static int processDiffPatch(
       }
     } else {
       if (side.equals(Side.LEFT)) {
-        return lineNumberToPositionLeftSide.get(number);
+        if (lineNumberToPositionLeftSide.containsKey(number)) {
+          return lineNumberToPositionLeftSide.get(number);
+          // It isn't possible to convert Reviewer's line number to GitHub patch(diff)'s position if
+          // the patch doesn't contain this line.
+        } else {
+          // XXX 1)How to continue the sync without exception.
```

Comment:
I would choose any line (maybe the closest, if that's possible), and add to the comment:
"Synced from line X"

Reply:
Added TODO for it.